### PR TITLE
Delete team as captain leave and remove player for captain

### LINKED ISSUE #4: Remove Player for Captain
Need to display team members (other than captain) in a dropdown for remove player.

- Only Captain can remove players from the team
- #3 needs to be resolved first

### PR DESCRIPTION
**Pull request by Garvit-32**
This PR fixes #4 and #6

### CODE REVIEW

File: registration/views.py [hunk 1]
```diff
@@ -75,5 +75,10 @@ def get_context_data(self, **kwargs):
         context = super(removePlayerView, self).get_context_data(**kwargs)
         user = get_object_or_404(UserProfile, user=self.request.user)
         users = UserProfile.objects.filter(teamId=user.teamId)
-        context['players'] = users
+        team = get_object_or_404(TeamRegistration, teamId=user.teamId)
+        userList = []
+        for i in users:
+            userList.append(i)
+        userList.remove(team.captian)
```
[the remark this answers is not in the document]
Comment (Garvit-32) [reply registration/views.py:82]: file changed as per request and dropdown added 
Annotation 2020-08-25 193011

[the remark this answers is not in the document]
Comment (harsh-not-haarsh) [reply registration/views.py:82]: Looks good, Do this as well:

- [ ] Show "first_name (Email)" in the dropdown
- [ ] Get rid of the email list that is printed on the page

[the remark this answers is not in the document]
Comment (harsh-not-haarsh) [reply registration/views.py:82]: Looks good, Do this as well:

- [ ]  Show "first_name (Email)" in the dropdown
- [ ] Get rid of the email list that is printed on the page

**Review by harsh-not-haarsh — CHANGES_REQUESTED** [review]

File: registration/views.py (hunk 1 above)
[the remark this answers is not in the document]
Comment (Garvit-32) [reply registration/views.py:82]: changes made as requested
Annotation 2020-08-26 071411

**Review by harsh-not-haarsh — APPROVED** [review]

**Pull request merged**